sevenseconds/cli.py

```diff
@@ -80,11 +80,12 @@ def configure(file, account_name_pattern, **options):
             file,
             account_name_pattern,
             options)
+        sessions = list(sessions.values())
```

Comment:
I find it ugly that the variable is initialized as hash and then overwritten as an array